herokuappTests/utility/request_utils.py

```diff
@@ -0,0 +1,26 @@
+import requests
+
+class RequestUtils:
+    baseURL = "https://restful-booker.herokuapp.com/"
```

Comment:
You don't need baseurl defined here. Itr should be available from constants like mentioned above.